index.html

```diff
@@ -41,9 +46,14 @@ <h3>Lodging</h3>
         <ul id="lodging-list" class="list-container">
             <li class="list-items">list will go here. </li>
         </ul>
+        </section> 
     </section>
 
-    <footer>Made by The Trail Lad's! &copy2021</footer>
+    
+
+    <footer class="">Made by The Trail Lad's! &copy2021</footer>
```

Comment:
Did you mean to leave an empty class declaration?

Reply:
I meant to delete it. I had left it blank on purpose when trying to figure out the flex boxes.